### PR TITLE
[charts-all] replace bitnami-shell with os-shell

### PR DESCRIPTION
**Pull request by deantchi**
- replace `bitnami-shell:11-debian-11-r87` with `os-shell:11-debian-11-r89`
- update all chart version and chart dip lock version

### CODE REVIEW

File: hirosystems/bitcoin-core/values.yaml [hunk 1]
```diff
@@ -653,8 +653,8 @@ volumePermissions:
   ##
   image:
     registry: docker.io
-    repository: bitnami/bitnami-shell
-    tag: 11-debian-11-r87
+    repository: bitnami/os-shell
+    tag: 11-debian-11-r89
```
Comment (CharlieC3) [inline hirosystems/bitcoin-core/values.yaml:657]: Any reason why `r89` was chosen? The latest tag is `11-debian-11-r110`, maybe we should update to the latest instead?

Comment (deantchi) [reply hirosystems/bitcoin-core/values.yaml:657]: I'm usually not a big fan of jumping so many version so I wanted to use a version as close as possible to the current version deployed.  If you think it's okay to use the latest and I can push the change.

Comment (CharlieC3) [reply hirosystems/bitcoin-core/values.yaml:657]: I'm pretty sure it's just security patches from the upstream distro base image, so it should be fine. But am also fine if you'd prefer not to.

Comment (deantchi) [reply hirosystems/bitcoin-core/values.yaml]: I just pushed a change to use the latest.

**Review by CharlieC3 — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**